style.css

```diff
@@ -62,3 +62,177 @@
   position: absolute;
   padding-top: 30px;
 }
+
+.works {
+  display: grid;
+  row-gap: 20px;
+  grid-template-rows: auto;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin-left: 28px;
+  width: 327px;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+#Recent-Works-Text {
+  text-align: center;
+  width: 327px;
+  height: 97px;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  padding-left: 25px;
+}
+
+.img-Plaholder {
+  background-color: #ebf0ee;
+  width: 327px;
+  height: 230px;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.project-detail {
+  width: 327px;
+  height: 222px;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+#card-text {
+  font-size: 24px;
+  font-weight: 600;
+  width: 295px;
+  height: 57px;
+  padding-left: 20px;
+  line-height: 32px;
+  text-align: center;
+}
+
+#work-list {
+  margin: 0;
+  padding-left: 20px;
+}
+
+#work-list li {
+  display: inline;
+  padding: 9px;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+}
+
+#see-project-btn {
+  width: 122px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background-color: #36b37f;
+  border-radius: 4px;
+  margin-left: 110px;
+  margin-top: 30px;
+}
+
+#see-project {
+  text-align: center;
+  padding-top: 12px;
+}
+
+.about-title {
+  display: flex;
+  justify-content: flex-end;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  height: 100px;
+}
+
+.about-detail {
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: center;
+}
+
+#about-text {
+  padding-right: 35px;
+}
+
+#get-my-resume-btn {
+  width: 150px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #fff;
+  background-color: #36b37f;
+  border-radius: 4px;
+  margin-left: 115px;
+  margin-top: 30px;
+}
+
+#get-my-resume {
+  text-align: center;
+  padding-top: 12px;
+}
+
+.category {
+  display: flex;
+  flex-direction: column;
+  width: 327px;
+  height: 352px;
+  background-color: #ebf0ee;
+  align-items: center;
+  margin-left: 30px;
+  border-radius: 8px;
+}
```

Comment:
Kindly note that the margins in your page are different from those in the figma design. I suggest you fix this

- [ ] Your design
<img width="540" alt="Screenshot 2022-09-24 at 10 38 50" src="https://user-images.githubusercontent.com/48546992/192086681-60cdd186-5b51-47c3-9e66-a7ee534dec46.png">

- [ ] Figma design
<img width="193" alt="Screenshot 2022-09-24 at 10 37 27" src="https://user-images.githubusercontent.com/48546992/192086700-104db2a9-34da-4656-882b-e720c097d2a2.png">

Reply:
- [ ] Kindly include this section too in your project. This section is missing